src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,12 @@
 // Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+export const UserInfo = ({ user }) => {
+  const { email, name } = user;
+
+  return (
+    <>
+      <a className="UserInfo" href={`mailto:${email}`}>
+        {name}
+      </a>
+    </>
+  );
+};
```

Comment:
You don't need so many lines
```suggestion
export const UserInfo = ({ user }) => (
  <a className="UserInfo" href={`mailto:${user.email}`}>
    {user.name}
  </a>
);
```